### DISCUSSION TITLE
ArchR support  for 3D Monocle trajectories

### DISCUSSION THREAD

**Discussion by dimurali93** [GreenleafLab/ArchR#1898] [Feature Requests]
Hi,

We would like to use Monocle3D trajectories with ArchR multiome pipeline for better visualization of trajectories. Is there away to do this in the current setup? If not, do you plan to support this feature in the near future.
Thanks,
Divya

**Comment by rcorces** [comment]
This is not currently planned. Happy to review a pull request to add this feature but we dont have bandwidth to do this ourselves at the moment.
